lib/apivore/validator.rb

```diff
@@ -21,8 +22,7 @@ def matches?(swagger_checker)
         send(
           method,
           full_path(swagger_checker),
-          params['_data'] || {},
-          params['_headers'] || {}
+          *RailsShim.request_params(params)
```

Comment:
I think it might be good to keep the selection and defaulting of values back here if possible so no logic beyond transforming the structure of the arguments is handled by the shim. Also, a more verbose method wouldn't hurt since we don't care for this method to be generic at this point.

Maybe something along the lines of:

```ruby
*RailsShim.active_dispatch_request_params(
  params: params['_data'] || {} , 
  headers: params['_header'] || {}
)
```

Or allow for all args to be passed through so this shim handles the entire set of args rather a subset.

```ruby
*RailsShim.active_dispatch_request_args(
  method, 
  full_path(swagger_checker),
  params: params['_data'] || {} , 
  headers: params['_header'] || {}
)
```

I personally favor the second.

@amcaplan @gwshaw